hydroshare/local_settings.py

```diff
@@ -9,6 +9,7 @@
 from kombu.common import Broadcast
 
 DEBUG = True
+TEMPLATE_DEBUG = True
```

Comment:
@pkdash @mauriel133 I don't think we should add this TEMPLATE_DEBUG setting in our code base here. It appears this is deprecated in django 1.9, in which DEBUG and TEMPLATE_DEBUG were consolidated into one single DEBUG setting. @mjstealey has to set DEBUG to False for production, so it is best if we don't introduce another variable to set differently between production and development.